test/mock/rtps/ReaderHistory/fastdds/rtps/history/ReaderHistory.h

```diff
@@ -72,6 +72,13 @@ class ReaderHistory
         return ret;
     }
 
+    virtual bool can_change_be_added_nts(
```

Comment:
No plan for a test were this funtion fails?

Reply:
I think it should be implicitly checked by some of the tests using KEEP_ALL policies